### PR TITLE
prepare upcoming release

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: DEVELOP.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+===============
+Developer Guide
+===============
+
+Setup
+=====
+
+This project uses buildout_ to set up the development environment.
```
Comment (mfussenegger) [inline DEVELOP.rst]: buildout?

File: CHANGES.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Unreleased
+==========
+
+- Use Postgres wire protocol (pgx client library) to connect to CrateDB:
+
+  - This change requires CrateDB 3.1.0 or newer!
+
+  - Connections can be configured via ``crate.yml`` configuration file using
+    the ``-config.file`` flag.
```
Comment (mfussenegger) [inline CHANGES.rst]: Out of scope for this PR, but wouldn't it be nicer to be able to specify a connection-string directly? Would be docker friendlier as one doesn't have to bundle the right `crate.yml`

File: DEVELOP.rst (hunk 1 above)
Comment (chaudum) [reply DEVELOP.rst]: Oops, copy-paste ...

File: CHANGES.rst (hunk 2 above)
Comment (chaudum) [reply CHANGES.rst]: Agree, that's a valid point. Please make an issue with a feature request.

**Comment by chaudum** [comment]
@joemoe @mfussenegger pushed a fixup. can you have another look, please?

**Pull request merged**